modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/pointneuronnetwork/v0.1.0.json

```diff
@@ -0,0 +1,65 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/simulation/pointneuronnetwork/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/entity/v1.0.0"
+  ],
+  "shapes": [
+    {
+      "@id": "this:PointNeuronNetworkShape",
+      "@type": "sh:NodeShape",
+      "label": "Point neuron network.",
+      "targetClass": "nsg:PointNeuronNetwork",
+      "nodekind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/entity/v1.0.0/shapes/EntityShape"
```

Comment:
This should also inherit from ModelInstanceShape

Reply:
Yes, probably it will. We'll have another round of reviews for this in two weeks, so this pull request will change, I guess.

Reply:
Done.